Common/Source/Dialogs/dlgProgress.h

```diff
@@ -14,6 +14,6 @@
 
 void CreateProgressDialog(const TCHAR *text);
 void CloseProgressDialog();
-
+void ProgressDialogText(const TCHAR* text);
```

Comment:
unused and useless

Reply:
all definitions are needed, what's unused in your eyes?